include/circt/Dialect/FIRRTL/FIRRTLOps.h

```diff
@@ -95,6 +95,11 @@ inline MemDirAttr &operator|=(MemDirAttr &lhs, MemDirAttr rhs) {
   return lhs;
 }
 
+/// Return the StringAttr for the inner_sym name, if it exists.
+inline StringAttr getInnerSymName(Operation *op) {
+  return op->getAttrOfType<StringAttr>("inner_sym");
```

Comment:
This would likely be better as:

```
return op->getAttrOfType<StringAttr>(InnerSymbolTable.getInnerSymbolNameAttr());
```

Reply:
Yep, I think it makes sense to push it into `InnerSymbolTable` while we're at it, instead of a floating function, since it's the canonical location for saying what the inner symbol attribute name is (as done in #3407, similar to `SymbolTable::getSymbolName`).  I was looking/planning to eventually move all/most mentions of the attribute and any manual manipulation of it, into `InnerSymbolTable` and related classes/interfaces/traits (or some common location anyway).